diff -urNp -x '*.orig' lame-3.100.org/configure.in lame-3.100/configure.in
--- lame-3.100.org/configure.in	2017-08-15 17:16:31.000000000 +0200
+++ lame-3.100/configure.in	2023-12-09 13:08:16.736805434 +0100
@@ -418,9 +418,10 @@ dnl Checks for libraries.
 
 AC_CHECK_HEADERS(termcap.h)
 AC_CHECK_HEADERS(ncurses/termcap.h)
-AC_CHECK_LIB(termcap, initscr, HAVE_TERMCAP="termcap")
-AC_CHECK_LIB(curses, initscr, HAVE_TERMCAP="curses")
-AC_CHECK_LIB(ncurses, initscr, HAVE_TERMCAP="ncurses")
+AC_CHECK_LIB(tinfo, tgetent, HAVE_TERMCAP="tinfo", [
+  AC_CHECK_LIB(ncurses, tgetent, HAVE_TERMCAP="ncurses", [
+    AC_CHECK_LIB(curses, tgetent, HAVE_TERMCAP="curses", [
+      AC_CHECK_LIB(termcap, tgetent, HAVE_TERMCAP="termcap")])])])
 
 AM_ICONV
 
