--- lame-398/frontend/Makefile.am.orig	2006-09-30 11:17:05.000000000 +0200
+++ lame-398/frontend/Makefile.am	2008-08-23 14:02:05.466113358 +0200
@@ -52,7 +52,7 @@
 endif
 
 CFLAGS = @CFLAGS@ @GTK_CFLAGS@ @FRONTEND_CFLAGS@ @SNDFILE_CFLAGS@
-LDFLAGS = @LDFLAGS@ @FRONTEND_LDFLAGS@ @SNDFILE_LIBS@ -static
+LDFLAGS = @LDFLAGS@ @FRONTEND_LDFLAGS@ @SNDFILE_LIBS@
 
 INCLUDES = -I$(top_srcdir)/libmp3lame -I$(top_srcdir)/include -I$(top_builddir)
 
--- lame-3.97/configure.in.orig	2006-09-24 15:53:31.000000000 +0200
+++ lame-3.97/configure.in	2006-09-26 20:31:27.993719750 +0200
@@ -347,9 +347,10 @@
 
 AC_CHECK_HEADERS(termcap.h)
 AC_CHECK_HEADERS(ncurses/termcap.h)
-AC_CHECK_LIB(termcap, initscr, HAVE_TERMCAP="termcap")
-AC_CHECK_LIB(curses, initscr, HAVE_TERMCAP="curses")
-AC_CHECK_LIB(ncurses, initscr, HAVE_TERMCAP="ncurses")
+AC_CHECK_LIB(tinfo, tgetent, HAVE_TERMCAP="tinfo", [
+  AC_CHECK_LIB(ncurses, tgetent, HAVE_TERMCAP="ncurses", [
+    AC_CHECK_LIB(curses, tgetent, HAVE_TERMCAP="curses", [
+      AC_CHECK_LIB(termcap, tgetent, HAVE_TERMCAP="termcap")])])])
  
 dnl math lib
 AC_CHECK_LIB(m, cos, USE_LIBM="-lm")
