--- lame-3.97/frontend/Makefile.am~	2006-07-14 14:26:41.000000000 +0200
+++ lame-3.97/frontend/Makefile.am	2006-07-14 14:29:39.000000000 +0200
@@ -35,9 +35,9 @@
 	portableio.h \
 	timestatus.h
 
-## EXTRA_lame__EXEEXT__SOURCES = $(brhist_sources)
-## EXTRA_mp3rtp__EXEEXT__SOURCES = $(brhist_sources)
-## EXTRA_mp3x__EXEEXT__SOURCES = $(brhist_sources)
+EXTRA_lame__EXEEXT__SOURCES = $(brhist_sources)
+EXTRA_mp3rtp__EXEEXT__SOURCES = $(brhist_sources)
+EXTRA_mp3x__EXEEXT__SOURCES = $(brhist_sources)
 
 if WITH_BRHIST
 lame_SOURCES = main.c $(common_sources) $(brhist_sources)
