--- lame-3.93.1/libmp3lame/Makefile.am.orig	Sun Nov 17 11:16:38 2002
+++ lame-3.93.1/libmp3lame/Makefile.am	Sat Jan 18 11:57:30 2003
@@ -18,7 +18,7 @@
 decoder_ldadd =
 endif
 
-libmp3lame_la_LIBADD = $(nasm_ldadd) $(decoder_ldadd)
+libmp3lame_la_LIBADD = $(nasm_ldadd) $(decoder_ldadd) -lm
 libmp3lame_la_LDFLAGS = -version-info @LIB_MAJOR_VERSION@:@LIB_MINOR_VERSION@ \
 			-no-undefined
 
--- lame-3.93.1/frontend/Makefile.am.orig	Sat Apr  6 16:08:18 2002
+++ lame-3.93.1/frontend/Makefile.am	Sat Jan 18 11:59:55 2003
@@ -53,7 +53,7 @@
 endif
 
 CFLAGS = @CFLAGS@ @GTK_CFLAGS@ @FRONTEND_CFLAGS@
-LDFLAGS = @LDFLAGS@ @FRONTEND_LDFLAGS@ -static
+LDFLAGS = @LDFLAGS@ @FRONTEND_LDFLAGS@
 
 INCLUDES = -I$(top_srcdir)/libmp3lame -I$(top_srcdir)/include -I$(top_builddir)
 
--- lame-3.97/configure.in.orig	2006-09-24 15:53:31.000000000 +0200
+++ lame-3.97/configure.in	2006-09-26 20:31:27.993719750 +0200
@@ -347,9 +347,10 @@
 
 AC_CHECK_HEADERS(termcap.h)
 AC_CHECK_HEADERS(ncurses/termcap.h)
-AC_CHECK_LIB(termcap, initscr, HAVE_TERMCAP="termcap")
-AC_CHECK_LIB(curses, initscr, HAVE_TERMCAP="curses")
-AC_CHECK_LIB(ncurses, initscr, HAVE_TERMCAP="ncurses")
+AC_CHECK_LIB(tinfo, tgetent, HAVE_TERMCAP="tinfo", [
+  AC_CHECK_LIB(ncurses, tgetent, HAVE_TERMCAP="ncurses", [
+    AC_CHECK_LIB(curses, tgetent, HAVE_TERMCAP="curses", [
+      AC_CHECK_LIB(termcap, tgetent, HAVE_TERMCAP="termcap")])])])
  
 dnl math lib
 AC_CHECK_LIB(m, cos, USE_LIBM="-lm")
